crossdock/otel-agent-config.yml

```diff
@@ -0,0 +1,28 @@
+receivers:
+  jaeger:
+    remote_sampling:
+      host_endpoint: localhost:5778
+      fetch_endpoint: jaeger-collector:14250
+    protocols:
+      grpc:
```

Comment:
Should grpc and thrift_http be here, if providing the role of the jaeger agent?

Reply:
I am not sure if we can disable `grpc` I think the sampling manager depends on it in the OTEL receiver. 

`thrift_http` can be disabled though.